scripts/sts/sts.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# USAGE
+#
+# ~ Creation ~
+#
+# make ocm/provision_sts (optional <CLUSTER_NAME= > <AWS_REGION= > <NODE_COUNT= > <MACHINE_TYPE= >)
+#
+# <Default values>
+#  CLUSTER_NAME=defaultsts
+#  AWS_REGION=eu-west-1
+#  NODE_COUN=4
+#  MACHINE_TYPE=m5.xlarge
+#
+# ^C to break
+# Create a STS cluster
+#
+# ~ Deletion ~
+#
+# make ocm/delete_sts <CLUSTER_NAME= > <AWS_REGION= > (optional <PREFIX= >)
+#
+# <Default values>
+#  PREFIX=ManagedOpenShift
+#  
+# ^C to break
+# Delete a STS cluster
+#
+# PREREQUISITES
+# - ROSA CLI
+# - AWS CLI
+# - aws configuration with valid permissions. Run `aws configure`
+#
+# MORE INFO
+# https://docs.openshift.com/rosa/rosa_getting_started_sts/rosa-sts-setting-up-environment.html
+
+set -eux
+aws_account_id=$(aws sts get-caller-identity | jq -r .Account)
+OCM_ENV="${OCM_ENV:-production}"
+LOCAL_RUN=${LOCAL_RUN:-true}
+CLUSTER_NAME="${CLUSTER_NAME:-defaultsts}"
+AWS_REGION="${AWS_REGION:-eu-west-1}"
+PREFIX="${PREFIX:-ManagedOpenShift}"
+NODES_COUNT="${NODES_COUNT:-4}"
+MACHINE_TYPE="${MACHINE_TYPE:-m5.xlarge}"
+provision_sts_cluster() {
```

Comment:
```suggestion
set -eux

OCM_ENV="${OCM_ENV:-production}"
LOCAL_RUN=${LOCAL_RUN:-true}
CLUSTER_NAME="${CLUSTER_NAME:-defaultsts}"
AWS_REGION="${AWS_REGION:-eu-west-1}"
PREFIX="${PREFIX:-ManagedOpenShift}"
NODES_COUNT="${NODES_COUNT:-4}"
MACHINE_TYPE="${MACHINE_TYPE:-m5.xlarge}"

provision_sts_cluster() {
```
`aws_account_id` doesn't seem to be used and would be good to add some line breaks to sepeate local variable and start of functions :thinking: